traffic_portal/app/src/common/modules/table/cacheGroupServers/TableCacheGroupServersController.js

```diff
@@ -17,11 +17,13 @@
  * under the License.
  */
 
-var TableCacheGroupsServersController = function(cacheGroup, servers, $controller, $scope, $state, $uibModal, cacheGroupService) {
+var TableCacheGroupsServersController = function(cacheGroup, servers, $controller, $scope, $state, $uibModal, cacheGroupService, serverUtils) {
 
 	// extends the TableServersController to inherit common methods
 	angular.extend(this, $controller('TableServersController', { servers: servers, $scope: $scope }));
 
+	$scope.serverNetInfo = new Map(servers.map(function(s){return [s.id, serverUtils.toLegacyIPInfo(s.interfaces)];}));
```

Comment:
you should be able to just put this in the `TableServersController` which is the parent controller to all of these controllers rather than duplicating it across all server controllers.

Reply:
also, i was unable to find this column in 

![image](https://user-images.githubusercontent.com/251272/84523048-e9733400-ac94-11ea-9a33-f7fb314e17dd.png)